src/data/find-wallet/real-wallet-data.ts

```diff
@@ -0,0 +1,567 @@
+const walletData = [
```

Comment:
Nitpick - I think this directory can just be /wallets/, if we think a directory is needed at all. I imagine this data will eventually be used in /wallets page & /get-eth page in addition to the /find-wallet page.

Reply:
Also: is the plan to go with a TS data file? I thought we were going with JSON format.

I'm potentially ok with either. Just want to make sure we're making an intentional decision. e.g. can this data file be queried via GraphQL data layer for performance advantages?